extension/content.js

```diff
@@ -207,6 +207,58 @@ function indentInput(el, size = 4) {
 	el.selectionEnd = el.selectionStart = selectionStart + indentationText.length;
 }
 
+function lineUpInput(el, down) {
+	el.focus();
+	const value = el.value;
+	const selectionStart = el.selectionStart;
+	const selectionEnd = el.selectionEnd;
+	const lines = value.split(/(\r?\n)/);
```

Comment:
Yes, actually. It preserves the newline characters as part of the array: 

```
["a", "\n", " \b", "\r\n"]
```

It causes the newline character(s) to be preserved as is. I could just normalize to `\n`. Not sure preserving the newline format is worth it.